tifeatures/dependencies.py

```diff
@@ -251,3 +251,19 @@ def sortby_query(
 ):
     """Sortby dependency."""
     return sortby
+
+
+def geom_col(
+    collection: TableLayer = Depends(CollectionParams),
```

Comment:
I think this will add twice the Collection parameter in the openapi, it feels weird to have it at two different places

Reply:
It shouldn't, the docs say that you should go ahead and duplicate dependencies and that it will only calculate them once.